app.vue

```
@@ -107,20 +102,26 @@ const route = useRoute().path;
+let flags = {};
+
+// fetch response from feature flags service api 
+await $fetch('/api/getFeatureFlagValues')
+    .then((response) => {
+        flags = response;
+        flagStore.flags = response;        
+    });
+
+// set up a global function to allow checking of flags
+app.appContext.config.globalProperties.checkFlag = (str) => {
+    if ((Object.keys(flags).length > 0 && flags.hasOwnProperty(str) && flags[str].enabled) || checkQueryString(str)) {
```

ESLint

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

(no-prototype-builtins)